adapters/Mintegral/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### [Version 7.4.3.0](https://dl.google.com/googleadmobadssdk/mediation/ios/mintegral/MintegralAdapter-7.4.3.0.zip)
 - Verified compatibility with Mintegral SDK version 7.4.3.
+- Added watermark support for bidding ads.
```

Comment:
Remove this line and add this above:
```
#### Next version
- Added watermark support for bidding ads.
```

Reply:
@joshh-devrel 
I don’t understand what you mean. It is true that Mintegral SDK version 7.4.3 already supports admob watermark.

Reply:
@Mintegral-Lee 
To clarify, Mintegral iOS adapter 7.4.3.0 is already live so this code change can't be included in it, but the next version.